syntax = "proto2";
import "goby/middleware/protobuf/app_config.proto";
import "goby/zeromq/protobuf/interprocess_config.proto";

package goby.zeromq.protobuf;

message LoggerConfig
{
    optional goby.middleware.protobuf.AppConfig app = 1;
    optional InterProcessPortalConfig interprocess = 2;

    required string log_dir = 3;

    optional string type_regex = 4 [default = ".*"];
    optional string group_regex = 5 [default = ".*"];

    repeated string load_shared_library = 10;
    
}
